doc/classes/String.xml

```diff
@@ -522,6 +535,21 @@
 				Returns [code]true[/code] if this string represents a well-formatted IPv4 or IPv6 address. This method considers [url=https://en.wikipedia.org/wiki/Reserved_IP_addresses]reserved IP addresses[/url] such as [code]"0.0.0.0"[/code] and [code]"ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"[/code] as valid.
 			</description>
 		</method>
+		<method name="is_valid_unicode_identifier" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if this string is a valid Unicode identifier.
+				[codeblock]
+				print("node_2d".is_valid_unicode_identifier())    # Prints true
+				print("TYPE_FLOAT".is_valid_unicode_identifier()) # Prints true
+				print("1st_method".is_valid_unicode_identifier()) # Prints false
+				print("MyMethod#2".is_valid_unicode_identifier()) # Prints false
+				print("体力".is_valid_unicode_identifier())       # Prints true
```

Comment:
```suggestion
				print("体力".is_valid_unicode_identifier())        # Prints true
```

Reply:
This depends on the font used for rendering. Usually, when using a monospace font, each Chinese character is two Latin characters wide. GitHub might show them as not aligned because it's using different fonts for them.